src/assets/mouth.svg

```diff
@@ -1,4 +0,0 @@
-<svg width="60" height="30" viewBox="0 0 60 30" fill="none">
```

Comment:
Noticed these SVGs are set up as animated Components on each solution (e.g. `Slimy/Mouth/index.tsx`).

Should we keep the source `.svg` as a reference for students? Perhaps in some sort of `reference/` folder. Or, together with the `svg-transformer`, is it the intention that students start out importing the `svg` file and move on to the component version once it's animated?

Reply:
Hmm, that's a good question. I don't think it is necessary to keep the `svg-transformer` library, but the `.svg` files are necessary for them to be able to copy and animate. 

Does it make sense to keep it on the repo though? I was thinking about deploying the training (gh pages or something like that) and maybe that's where the students can get the source `.svg` files from, and this repo can be more of a solution/showcase of custom challenges created by former trainees. What do you think?

Reply:
Answering both SVG threads here.

I love the idea of a "training guide" of sorts, hosted. I think it's possible to host a gh-pages from this same repo. We could even have a cli/script to create a "clean slate" version that has just the basic box for slimy, and perhaps some dependencies. Interesting backlog :smile: 

In the meantime, I'll bring back the svg files.